core/app/api/customer/group/route.ts

```diff
@@ -0,0 +1,16 @@
+import { NextResponse } from 'next/server';
+
+import { getCustomerGroupId, GetCustomerGroupResponse } from '~/client/queries/get-customer';
+
+export const GET = async (): Promise<NextResponse<GetCustomerGroupResponse>> => {
+  try {
+    const { data } = await getCustomerGroupId();
+
+    return NextResponse.json(data);
+  } catch (error) {
+    const errorMessage =
+      error instanceof Error ? error.message : 'Something went wrong fetching the customer group.';
+
+    return NextResponse.json({ customer: null, error: errorMessage }, { status: 500 });
+  }
```

Comment:
The try/catch seems unnecessary here. The message, "Something went wrong fetching the customer group", is somewhat obvious if we get a 500: something went wrong sending a GET to get customer groups. This function can probably just be this:

```ts
export async function GET() {
  const { data } = await getCustomerGroupId();

  return NextResponse.json(data);
}
```

What, other than `data`, does the function return?